tools/build.js

```diff
@@ -578,6 +581,11 @@ function buildAll(options) {
             callback: publish,
             enabled: options.publish,
         },
+        {
+            message: 'Publishing dark mode variants to npm...',
+            callback: () => publish(true),
```

Comment:
Instead of passing true here, how about pass the tag name to make the publish function more reusable?